arbitrum-docs/partials/_contribute-docs-partial.md

```diff
@@ -0,0 +1,258 @@
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md'; 
+
+:::tip Submitting a docs PR?
+
+**Please review this guidance before issuing pull requests** into the [Arbitrum docs repository](https://developer.arbitrum.io/). This guidance helps us **streamline** and **scale** our efforts. Thank you!
+
+:::
+
+<PublicPreviewBannerPartial />
+
+The [`developer.arbitrum.io`](https://developer.arbitrum.io/) docs portal is the **single source of truth** for documentation that supports Offchain's product portfolio. This includes documentation for:
+
+1. [Building dApps](/quickstart-solidity-hardhat) with Arbitrum’s chains.
+2. [Bridging tokens](../getting-started-users.mdx) to Arbitrum’s chains.
+3. [Running an Arbitrum node](../node-running/quickstart-running-a-node.md).
+4. [Launching a self-managed chain](../launch-orbit-chain/orbit-quickstart.md) using Arbitrum Orbit.
+5. [Educational materials](../intro) that explain how these technologies work.
+
+Contributions to each of these content areas are welcome from the entire Ethereum community.
```

Comment:
```suggestion
Contributions to each of these content areas are welcome from the entire Arbitrum community.
```

Not sure if you intended to specify Ethereum here for a reason, but flaggin just in case.

Reply:
Ethereum was intentional :). We're an extension of that community, in my mind, and want to invite participation from "root and all".